modules/oe_translation_cdt/modules/oe_translation_cdt_mock/oe_translation_cdt_mock.install

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
I also added this change to make testing a bit more reliable, previously the mock module would throw an error if the old token was still valid.